openapi/index.yaml

```diff
@@ -40,10 +40,60 @@ paths:
           description: Not found.
       parameters: []
 definitions:
+  Timestamp:
+    $ref: "https://raw.githubusercontent.com/pagopa/io-functions-commons/v10.2.3/openapi/definitions.yaml#/Timestamp"
+  FiscalCode:
+    $ref: "https://raw.githubusercontent.com/pagopa/io-functions-commons/v10.2.3/openapi/definitions.yaml#/FiscalCode"
+  ProblemJson:
+    $ref: "https://raw.githubusercontent.com/pagopa/io-functions-commons/v10.2.3/openapi/definitions.yaml#/ProblemJson"
+
   Input:
     type: object
     properties:
       id:
         type: string
       output:
         type: string
+  CgnPendingStatus:
```

Comment:
why not just a `CngStatus` with _all_ the enum values?

Reply:
That's because i will add some specific attributes to each status into an other PR so i need that `status` field is fixed to the correct value. It's something similar to (https://github.com/pagopa/io-functions-bonus/blob/da0fbfa011850a1bd64bab2384cf6702cb69517c/openapi/index.yaml#L340)[https://github.com/pagopa/io-functions-bonus/blob/da0fbfa011850a1bd64bab2384cf6702cb69517c/openapi/index.yaml#L340]